plugins/livelog/livelog.go

```diff
@@ -100,11 +103,42 @@ func (tp *taskPlugin) Finished(success bool) error {
 		return err
 	}
 	defer file.Close()
+
+	tempDir, err := runtime.NewTemporaryStorage(os.TempDir())
+	if err != nil {
+		return err
+	}
+
+	tempFile, err := tempDir.NewFile()
+	if err != nil {
+		return err
+	}
+
+	defer tempFile.Close()
```

Comment:
If you really wanted to log the error, which honestly isn't critical, but if you wanted to... you would use the `Log` from `TaskPluginOptions`, keep a reference to the `options.Log` from `NewTaskPlugin`